src/openloco/openloco.cpp

```diff
@@ -164,6 +165,15 @@ namespace openloco
         return paused_state;
     }
 
+    // 0x00431E32
+    // value: bl (false will no-op, seems pointless to me)
```

Comment:
This is a game action, right? `bl` is the apply flag.

Reply:
ooooooh, right ok